Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocCheckBox.css

```
@@ -2,39 +2,63 @@
 
 .bocCheckBox
 {
-  margin-right: 0.5em;
+  display: inline-flex;
+  align-items: center;
+}
+
+.bocCheckBox input
+{
+  position: absolute;
+  z-index: -1;
+}
+
+.bocCheckBox span.description
+{
+  display: inline-flex;
+  align-items: center;
+}
+
+.bocCheckBox img,
+.bocCheckBox input ~ span.image-visualizer::before
```

```suggestion
.bocCheckBox input + span.image-visualizer::before
```
Hmm... at this point, the image visualizer is rendered immediately after the input element and there's no reason to change this so using `+` is more expressive to me. Thoughts?

(and I didn't patch the other occurances at this time)